test/forms/inputs/dropdown-checkbox-group.test.js

```diff
@@ -62,3 +61,19 @@ test('DropdownCheckboxGroup removes value from array when selected option clicke
   const selectValueLabel = screen.getByText('None')
   expect(selectValueLabel).toBeInTheDocument()
 })
+
+test('DropdownCheckboxGroup sets menu no longer active when clicked outside', async () => {
+  const user = userEvent.setup()
+
+  render(<WrappedDropdownCheckboxGroup value={['1']} />)
+
+  const select = screen.getByRole('button')
+  await act(() => user.click(select))
```

Comment:
Why is `act` necessary here?

Reply:
My understanding is that any code that relies on asserting on state changes within a component should be wrapped in `act`. The button's `nextSibling` will only have the `is-active` class applied if the internal `expanded` state changes

Reply:
Hmm, it looks like all events within React's callstack should be covered, so maybe I don't need it here!

https://kentcdodds.com/blog/fix-the-not-wrapped-in-act-warning

Additionally, it's possible that instead of calling `act` directly, I should be using `waitFor` (https://testing-library.com/docs/react-testing-library/cheatsheet/#async). What do you think?

Reply:
Ok, fixed it :)